cmd/osbuild-worker-executor/handler_build.go

```diff
@@ -188,8 +189,8 @@ func handleIncludedSources(atar *tar.Reader, buildDir string) error {
 		if filepath.Clean(name) != strings.TrimSuffix(name, "/") {
 			return fmt.Errorf("name not clean: %v != %v", filepath.Clean(name), name)
 		}
-		if !strings.HasPrefix(name, "store/") {
-			return fmt.Errorf("expected store/ prefix, got %v", hdr.Name)
+		if !strings.HasPrefix(hdr.Name, "osbuild-store/") {
```

Comment:
This should probably be `hdr.Name` -> `name` for consistency with the lines around it.

Reply:
oh right, rebase fallout